pkg/crc/validation/validation.go

```diff
@@ -58,16 +59,22 @@ func ValidateEnoughMemory(value int) error {
 }
 
 // ValidateBundlePath checks if the provided bundle path exist
-func ValidateBundlePath(bundle string) error {
-	if err := ValidatePath(bundle); err != nil {
+func ValidateBundlePath(bundlePath string) error {
+	if err := ValidatePath(bundlePath); err != nil {
 		if constants.BundleEmbedded() {
 			return fmt.Errorf("Run 'crc setup' to unpack the bundle to disk")
 		}
-		return fmt.Errorf("%s not found, please provide the path to a valid bundle using the -b option", bundle)
+		return fmt.Errorf("%s not found, please provide the path to a valid bundle using the -b option", bundlePath)
 	}
 
-	userProvidedBundle := filepath.Base(bundle)
+	userProvidedBundle := filepath.Base(bundlePath)
 	if userProvidedBundle != constants.GetDefaultBundle() {
+		usingCustomBundle := strings.HasPrefix(bundle.GetBundleNameWithoutExtension(userProvidedBundle),
+			bundle.GetBundleNameWithoutExtension(constants.GetDefaultBundle()))
```

Comment:
I realized it would be better to append a `_` here, as we don't want `crc_libvirt_4.7.15.crcbundle` to be detected as a custom bundle for `crc_libvirt_4.7.1.crcbundle` :)

Reply:
@cfergeau I knew it something will be come up at the last moment in your mind so didn't merge it :P

Reply:
I don't think it's really important to be honest, can easily be fixed afterwards :)